### ISSUE TITLE
Allow Lacona to control Spotify

### ISSUE THREAD

**Issue by brandonhorst** [laconalabs/LaconaApp#9]
It's likely that more people use Spotify than iTunes. I'll have to look into the AppleScript libraries and see what is possible.

**Comment by yadomi** [comment]
Maybe it's would be better to just send specific events just like keyboard media keys ? it's more generic than just Spotify

**Comment by jethroguce** [comment]
👍

**Comment by mlumain** [comment]
Is this currently working on Lacona?

**Comment by brandonhorst** [comment]
No. I have a version here: https://github.com/brandonhorst/lacona-spotify

However, all that it can do is search Spotify for music and play it - it can't interact with your local playlists or stored music, it doesn't work for people outside the US, and it's really slow.

I want to continue making improvements. If you want to join in, you can try it out with `git clone` and then `lacona install`.
